websites/H/HackTheBox/presence.ts

```diff
@@ -0,0 +1,215 @@
+const presences = [
```

Comment:
This should be a Record<string, PresenceData> instead
https://github.com/PreMiD/Presences/blob/236523980322a4053a45c691a1739eb50e2aea48/websites/G/Geotastic/presence.ts#L12-L42

Reply:
Yeah, that's great, but it needs to be somehow extended by the stateMethod I have there.